test/reachsets.jl

```diff
@@ -101,5 +101,36 @@ end
     @test tspan(c) == tspan(d) == 0..1
     @test isequivalent(set(overapproximate(c, Zonotope)), set(R))
     @test isequivalent(set(overapproximate(d, Zonotope)), set(R))
+                    
+end
 
+@testset "Taylor model reach-sets with non-float coefficients" begin
+    
+    using TaylorModels:Taylor1
```

Comment:
this is not the only missing name (see [here](https://github.com/JuliaReach/ReachabilityAnalysis.jl/runs/6047778975?check_suite_focus=true#step:6:620), note it only shows the next missing one) maybe try to run the tests locally first